### PR TITLE
feat(mode): Warning mode

### PR DESCRIPTION
**Pull request by mvaldes42**
- adds a warning mode with option `--onlyWarnings`
- changes output format to avoid unnecessary `\n`
- rm changelog from repository as the repository is public

### CODE REVIEW

File: checkForRelease.js [hunk 1]
```diff
@@ -202,24 +223,37 @@ function formatLink(key) {
...
-  let subTasks = await formatSubtasks(issue);
-  return formatUS(issue) + subTasks + formatLink(issue.key);
+  const usFormatted = formatUS(issue);
+  let subFormatted = "";
+  if (usFormatted !== "") subFormatted = await formatSubtasks(issue);
+  const result =
+    usFormatted +
+    subFormatted +
+    `${usFormatted === "" ? "" : formatLink(issue.key)}`;
+  debugger;
```
Comment (bryneTane) [inline checkForRelease.js:233]: Debugger qui traine

File: checkForRelease.js [hunk 2]
```diff
@@ -159,20 +168,32 @@ async function formatSingleSubtask(sub) {
...
+
+  if (options && options.onlyWarnings) {
+    if (!isUsReady && !isUsInProd)
+      return (
+        `${modeBold}${colorNotReady}[❌ ${issue.fields.status.name}]` +
+        ` (${issue.key}) @` +
+        issue.fields.creator.displayName +
+        ` ${issue.fields.summary}${modeEscape}`
+      );
+    else return "";
```
Comment (bryneTane) [inline checkForRelease.js:195]: Else !

File: checkForRelease.js [hunk 3]
```diff
@@ -144,6 +144,15 @@ async function formatSingleSubtask(sub) {
       assigneeName = "no-assignee";
     }
   }
+  if (options && options.onlyWarnings) {
+    if (!isReady && !isProd)
+      return (
+        modeDim +
+        `${colorNotReady}(👎 ${sub.fields.status.name} @${assigneeName} ${sub.key})` +
```
Comment (bryneTane) [inline checkForRelease.js:151]: Autant passer les variables concaténées dans la template string aussi

File: checkForRelease.js (hunk 1 above)
Comment (mvaldes42) [reply checkForRelease.js]: 👍

File: checkForRelease.js (hunk 2 above)
Comment (mvaldes42) [reply checkForRelease.js]: 👍

File: checkForRelease.js (hunk 3 above)
Comment (mvaldes42) [reply checkForRelease.js]: 👍

**Pull request merged**